README.md

```diff
@@ -40,13 +42,66 @@ Configure `/etc/readyset_proxysql_scheduler.cnf` as follow:
 * `proxysql_port` - (Required) - Proxysql admin port
 * `readyset_user` - (Required) - Readyset application user
 * `readyset_password` - (Required) - Readyset application password
-* `readyset_host` - (Required) - Readyset host
-* `readyset_port` - (Required) - Readyset port
 * `source_hostgroup` - (Required) - Hostgroup running your Read workload
 * `readyset_hostgroup` - (Required) - Hostgroup where Readyset is configure
 * `warmup_time` - (Optional) - Time in seconds to mirror a query supported before redirecting the query to Readyset (Default 0 - no mirror)
 * `lock_file` - (Optional) - Lock file to prevent two instances of the scheduler to run at the same time (Default '/etc/readyset_scheduler.lock')
+* `operation_mode` - (Optional) - Operation mode to run the scheduler. The options are described in [Operation Mode](#operation-mode) (Default All).
+* `number_of_queries` - (Optional) - Number of queries to cache in Readyset (Default 10).
+* `query_discovery_mode` / `query_discovery_min_execution` / `query_discovery_min_row_sent` - (Optional) - Query Discovery configurations. The options are described in [Query Discovery](#query-discovery) (Default CountStar / 0 / 0).
+
+
+# Query Discovery
+The Query Discovery is a set of configuration to find queries that are supported by Readyset. The configurations are defined by the following fields:
+
+* `query_discovery_mode`: (Optional) - Mode to discover queries to automatically cache in Readyset. The options are described in [Query Discovery Mode](#query-discovery-mode)   (Default CountStar).
+* `query_discovery_min_execution`: (Optional) - Minimum number of executions of a query to be considered a candidate to be cached (Default 0).
+* `query_discovery_min_row_sent`: (Optional) - Minimum number of rows sent by a query to be considered a candidate to be cached (Default 0).
+
+# Query Discovery Mode
+The Query Discovery Mode is a set of possible rules to discover queries to automatically cache in Readyset. The options are:
+
+1. `CountStar` - Total Number of Query Executions
+ * Formula: `total_executions = count_star`
+ * Description: This metric gives the total number of times the query has been executed. It is valuable for understanding how frequently the query runs. A high count_star value suggests that the query is executed often.
+
+2. `SumTime` - Total Time Spent Executing the Query
+ * Formula: `total_execution_time = sum_time`
+ * Description: This metric represents the total cumulative time spent executing the query across all its executions. It provides a clear understanding of how much processing time the query is consuming over time. A high total execution time can indicate that the query is either frequently executed or is time-intensive to process.
```

Comment:
What are the units of time for all of these?

Reply:
microseconds

Reply:
Can we put that unit in the variable name?  Or if not the name, at least in the documentation here?

Reply:
done